packages/smart-account/src/index.ts

```diff
@@ -0,0 +1,17 @@
+import { Executor__factory } from './contracts';
+import { getConfig, getExecutor, getExecutorId } from './configs';
+
+export function getSmartIds(chainId: number) {
+  return getExecutorId(chainId);
+}
+
+export function verifySmartId(chainId: number, id: string) {
```

Comment:
```suggestion
export function isSupportedSmartAccountId(chainId: number, id: string) {
```
Prefer to keep the naming consistent with other npm in this repo such as `isSupportedChainId` and `isSupportedNetworkId`